test/api/portfolio_evidence_mailer_test.rb

```diff
@@ -0,0 +1,101 @@
+require 'test_helper'
+
+class PortfolioEvidenceMailerTest < ActionMailer::TestCase
+
+  #
+  #
+  # Init params for send mail function
+  #
+  #
+  setup do
+    @project = Project.first
+    task_definition = TaskDefinition.first
+    @tasks = []
+    @tasks << Task.create!(task_status_id: 1, task_definition_id: task_definition.id, project_id: @project.id)
+  end
+
+  #
+  # Send an email from main tutor to student in a project when having submitted tasks fail
+  #
+  test 'task_pdf_failed' do
+    email = PortfolioEvidenceMailer.task_pdf_failed(@project, @tasks)
+    assert_emails 1 do
+      email.deliver_now
+    end
+    assert_equal [@project.main_tutor.email], email.from
+    assert_equal [@project.student.email], email.to
+    assert_equal "#{@project.unit.name}: Task PDFs ready to view", email.subject
```

Comment:
Is this the right message to get when a task pdf fails to be created? I am sure I mentioned this earlier...